configs/example

```diff
@@ -0,0 +1,9 @@
+# the Docker repo for the bot image. Should be writable for you.
+DOCKER_REPO = <your-docker-user>/k8s-publishing-bot
```

Comment:
s/docker-user/docker-repo/

Reply:
actually it's `<docker-user>`, i.e. `<your-docker-user>/k8s-publishing-bot` is the repo, isn't it?

Reply:
nmind, repo says 'server' to me, not 'user'.